data/BeyondEternal.json

```
@@ -0,0 +1,27 @@
+{ 
+  "name": "Mario Solís", 
+ "type": "personal", 
+ "bio": "Software Engineer student and developer enjoyer" ,
+  "links": [ 
+    {
+      "name": "My GitHub",
+      "url": "https://github.com/BeyondEternal",
+      "icon": "FaGitHub"
```

```suggestion
      "icon": "FaGithub"
```

For the full list of icons see: https://linkfree.eddiehub.io/icons